docs/source/en/api/pipelines/deepfloyd_if.mdx

```diff
@@ -10,7 +10,7 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# IF
+# DeepFloyd IF 
```

Comment:
Let's keep this change and if you can also update it in the `_toctree.yml` that'd be great!